lib/components.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const path = require('path');
+const log = require('./logger')('nodecg/lib/components');
+const Bundles = require('./bundle-manager');
+
+log.trace('Adding Express routes for bundle components');
+
+const all = Bundles.all();
+
+for (let i = 0; i < all.length; i++) {
+	const bundle = all[i];
+	if (bundle.mount) {
+		for (let j = 0; j < bundle.mount.length; j++) {
+			const mountdir = bundle.mount[j];
+			if (typeof mountdir.endpoint !== 'string' || typeof mountdir.directory !== 'string') {
```

Comment:
This validation (and any additional validation) should go somewhere in [`lib/bundle-parser`](https://github.com/nodecg/nodecg/tree/master/lib/bundle-parser). I'd probably make a new file for it called `mount.js` or `mounts.js`.